adapters/wazirx-nft-Xauction-bsc/index.js

```diff
@@ -0,0 +1,162 @@
+require("dotenv").config();
+
+const web3 = require("web3");
+const moment = require("moment");
+const BigNumber = require("bignumber.js");
+const BSC = require("../../sdk/EVMC");
+const axios = require("axios");
+const URL = "http://nft-sales-service.dappradar.com/open-source";
+const TRANSFER_TOPIC = "0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef";
+const KEY = process.env.DAPPRADAR_API_KEY;
+const path = require("path");
+const { AVAILABLE_PROTOCOLS } = require("../../sdk/constants");
+
+const WRX_TOKEN = "0x8e17ed70334c87ece574c9d537bc153d8609e2a3";
+const BNB_TOKEN = "bnb";
+
+class WazirxNft {
+    constructor() {
+        this.name = "WazirXNFT";
+        this.symbol = "BNB";
+        this.token = BNB_TOKEN;
+        this.protocol = "bsc";
+        this.block = 14675841;
+        this.nftContract = "0x23cad0003e3a2b27b12359b25c25dd9a890af8e1";
+        this.contract = "0x12e2ea4adb0c384fb273279c9e169e2f494cca21";
+        this.events = ["AuctionFinalized"];
+        this.pathToAbi = path.join(__dirname, "./abi.json");
+        this.range = 500;
```

Comment:
I believe this should be `this.blockRange = 500` instead of `this.range = 500`

Reply:
This is an unused variable. Will be removing it.